pymongo/srv_resolver.py

```diff
@@ -15,17 +15,19 @@
 """Support for resolving hosts and options from mongodb+srv:// URIs."""
 
 import ipaddress
+import random
 
 try:
     from dns import resolver
     _HAVE_DNSPYTHON = True
 except ImportError:
     _HAVE_DNSPYTHON = False
 
+from random import sample
```

Comment:
This import is not needed since you use `random.sample` below.

Reply:
Done.